### PR TITLE
Added Slack Link in Header Fixes #1860

### LINKED ISSUE #1860: Link to community Slack
Would be nice to link to the Slack community channel. I feel like it falls into a similar category as the StackExchange or Reddit.

### PR DESCRIPTION
**Pull request by coderlcd**
Fixes #1860 

### Changes Summary

- Only modification is the addition of the Slack Icon and Link to the self-invite page for the Elementary Community Slack Page

This pull request is ready for review.

### CODE REVIEW

File: _templates/header.php [hunk 1]
```diff
@@ -150,6 +150,7 @@
                     <li><a href="https://www.reddit.com/r/elementaryos" target="_blank" rel="noopener" data-l10n-off title="Reddit"><i class="fa fa-reddit"></i></a></li>
                     <li><a href="https://elementaryos.stackexchange.com" target="_blank" rel="noopener" data-l10n-off title="StackExchange"><i class="fa fa-stack-exchange"></i></a></li>
                     <li><a href="https://twitter.com/elementary" target="_blank" rel="noopener" data-l10n-off title="Twitter"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="http://bit.ly/2jb31NO" target="_blank" rel="noopener" data-l10n-off title="Slack"><i class="fa fa-slack"></i></a></li>
```
Comment (lewisgoddard) [inline _templates/header.php]: I don't think we want to use a shortened link here.

Comment (lewisgoddard) [reply _templates/header.php]: Use the full https://elementarycommunity.slack.com/join/shared_invite/enQtMjg3NTExNDIwOTQ1LWExZjEzZGE4YjY4MjUwMTQ1MTVjZDJlNTFhNGRlN2FmZmUxNjhmZTI1YmNmZjQ0ODMxYzcwZTI1OTVlMTQyNWI instead.

**Comment by coderlcd** [comment]
I was just thinking about that when I was staring at the bit.ly link. I’ll get it switched over and resubmit. Thanks for the review.

File: _templates/header.php (hunk 1 above)
Comment (coderlcd) [reply _templates/header.php]: Thanks for the new link

**Comment by coderlcd** [comment]
Updated link!

**Comment by lewisgoddard** [comment]
@elementary/ux Do we want to use `slack` or `slack-hash` from FontAwesome?

https://fontawesome.com/icons?d=gallery&q=slack

**Comment by coderlcd** [comment]
I can update the link with either option once we get a decision! @lewisgoddard let me know if you hear one way or another.

**Comment by danrabbit** [comment]
`slack-hash` probably fits in better with the other icons

**Comment by cassidyjames** [comment]
Slack's brand guidelines say to use the bounded one if it's monochrome https://slack.com/brand-guidelines

**Pull request merged**